src/locale/locales/id/messages.po

```diff
@@ -4468,11 +4468,11 @@ msgstr "Tulis balasan Anda"
 
 #: src/screens/Onboarding/index.tsx:28
 msgid "Writers"
-msgstr ""
+msgstr "Sastra"
```

Comment:
```suggestion
msgstr "Penulis"
```
Would it be better if "Writers" is translated to simply "penulis" as not all writers are literary writers?

Reply:
What about the word `Sastrawan`?